spinel/const.py

```diff
@@ -401,6 +401,23 @@ class SPINEL(object):
     #    'S': ParentChanges                 (The number of times device changed its parents).
     PROP_CNTR_MLE_COUNTERS = PROP_CNTR__BEGIN + 402
 
+    # MAC retry histogram.
+    # Format: t(A(L))t(A(L)) (Read-only)
+    #
+    # The first structure is histogram which corresponds to retries of direct transmission:
+    #   'L': DirectRetry[0]                   (The number of packet afters 1 retry).
```

Comment:
After the first element will represent the 0 retry value, maybe the description could be as follow (or please align with .h version)?

```suggestion
    #   'L': DirectRetry[0]                   (The number of packets send with 0 retransmissions).
```